starter-code/pizza.js

```diff
@@ -33,53 +48,120 @@ function renderEverything() {
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll('.pep').forEach(function ($pep) {
     if (state.pepperonni) {
       $pep.style.visibility = "visible";
-    }
-    else {
+    } else {
       $pep.style.visibility = "hidden";
     }
   })
 }
 
 function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(function ($mush) {
+    if (state.mushrooms) {
+      $mush.style.visibility = "visible";
+    } else {
+      $mush.style.visibility = "hidden";
+    }
+  })
 }
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(function ($green) {
+    if (state.greenPeppers) {
+      $green.style.visibility = "visible";
+    } else {
+      $green.style.visibility = "hidden";
+    }
+  })
 }
 
+
 function renderWhiteSauce() {
-  // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll('.sauce').forEach(function ($sauce) {
+    if (state.whiteSauce) {
+      $sauce.style.visibility = "visible";
+    } else {
+      $sauce.style.visibility = "hidden";
+    }
+  })
 }
 
 function renderGlutenFreeCrust() {
-  // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll('.crust').forEach(function ($crust) {
+    if (state.glutenFreeCrust) {
+      $crust.style.visibility = "visible";
+    } else {
+      $crust.style.visibility = "hidden";
+    }
+  })
 }
 
+// get keys to use for matching the indices by extracting keys from state-object
+let ingredientKeys = Object.keys(state);
+
 function renderButtons() {
-  // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+  // based on index being identical for both the buttons and their corresponding position in the state-object, add/remove .active according to state being true/false
+  document.querySelectorAll('.btn').forEach(function (ingredient, index) {
+    if (state[ingredientKeys[index]]) {
+      ingredient.classList.add("active");
+    } else {
+      ingredient.classList.remove("active");
+    }
+  })
 }
 
+
 function renderPrice() {
-  // Iteration 4: change the HTML of `<aside class="panel price">`
+  // base price for pizza is 10
+  let totalPrice = 10;
```

Comment:
here you could assign totalPrice to basePrice (see the top of js file). Its always good not to hardcode things so you don't need to go back to your code to change all the values.  If you define it ones at the top and then refer to basePrice it is much simpler for you to change things in the future.